docs/templating/README.md

```diff
@@ -100,15 +100,15 @@ export async function getServerSideProps(
 ): Promise<void>;
 ```
 
-These functions can be used to make additional requests using the Apollo Client that will be executed when Next.js [Data Fetchers](https://nextjs.org/docs/basic-features/data-fetching) run.
+Use these functions to make additional requests using the Apollo Client, which executes with Next.js [Data Fetchers](https://nextjs.org/docs/basic-features/data-fetching) run.
 
-You may find this useful if you find that you need additional data from the WordPress backend that isn't otherwise fetched by default.
+You may find this helpful when you need additional data from the WordPress backend that does not fetch by default.
```

Comment:
@DonNateR How do you feel about this?

```suggestion
You may find this helpful when you need additional data from the WordPress backend that is not present by default.
```

Or just:

```suggestion
You may find this helpful when you need additional data from the WordPress backend.
```


`does not fetch` reads strangely in this context because it's the request that is fetching the data and not the WordPress backend.

Thank you for all your time reviewing our copy! I like the improvements here.